docs/source/cli/gee.rst

```diff
@@ -1,41 +1,41 @@
 Google Earth Engine
 ===================
 
-The earthengine tool is a utility program that allows you to manage Earth Engine assets and tasks from the command line. It is installed automatically when you install the Python API. To check whether the tool is installed and functioning correctly, type the following on a command line:
+The **earthengine** tool is a utility program that allows you to manage Google Earth Engine (GEE) assets and tasks from the command line. It is installed automatically when you install the Python API. To check whether the tool is installed and functioning correctly, enter the following in a command line:
 
 .. code-block:: console
 
     $ earthengine
 
-If the tool is properly installed then it will print out a short summary of available commands. To get help on a specific command, use:
+If the tool is properly installed, it will print out a short summary of available commands. To get help on a specific command, use:
 
 .. code-block:: console
 
     $ earthengine command -h
 
-When you first install the Python API you will need to log in using the :code:`authenticate` command described below.
+When you first install the Python API, you will need to login using the :code:`Authenticate` command described below.
```

Comment:
@alexryangregor Most of the:code: tags are related to a programming command, in this case, the command was in lowercase and therefore it has to maintain the syntax,